readme/product.md

```diff
@@ -2,9 +2,18 @@
 
 The `product` subfolder allows to import products provided as CSV files. See the example below:
 
-| id | name | type | product_variant_ids/categ_id/id 
-| - | - | - | - | - | - | - | - | - | - | - | - |
-| init.testProduct | My Product | service | product.product_category_all
+| id | product_variant_ids/id | 
+| - | - | 
+| init.testProduct | init.testProduct
```

Comment:
Do a product and its variants must share the same id?
eg, `init.testProduct`

Reply:
No, They don't have to